src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -113,8 +118,53 @@ void NonClientIslandWindow::OnSize(const UINT width, const UINT height)
         _rootGrid.Arrange(finalRect);
     }
 
+    winrt::check_bool(SetWindowPos(_interopWindowHandle, HWND_TOP, xPos, yPos, windowsWidth, windowsHeight, SWP_SHOWWINDOW));

```

Comment:
>HWND_TOP [](start = 57, length = 8)

Just double checking, you changed this from `HWND_BOTTOM` to `HWND_TOP`. Intentional?

Reply:
It appears that change is actually unnecessary, so I'll revert it.